code/modules/reagents/reagent_containers.dm

```diff
@@ -8,6 +8,10 @@
 	var/possible_transfer_amounts = list(5,10,15,25,30)
 	var/volume = 30
 	var/accuracy = 1
+	var/fragile = 2        // most glassware is super fragile. Not a boolean.
+	var/no_shatter = TRUE //does this container shatter?
```

Comment:
i think you should rename this variable to shatter instead and flip the true/false since i feel a double negative will be a little confusing